crashsimilarity/cache.py

```diff
@@ -0,0 +1,89 @@
+import json
+import pickle
+from abc import abstractmethod
+
+import logging
+
+from crashsimilarity import utils
+
+
+class BaseCache(object):
+    def __init__(self, name, file_name=None):
```

Comment:
OK, let's remove the option to keep the code simple. We can add the option if we actually need it.